lib/core/cardano-wallet-core.cabal

```diff
@@ -191,8 +191,10 @@ benchmark db
     , containers
     , cryptonite
     , deepseq
+    , directory
     , fmt
     , memory
+    , temporary
```

Comment:
Note that there's no need for an extra dependency here, we already have what we need from the `base` package:

https://hackage.haskell.org/package/base-4.11.1.0/docs/System-IO.html#v:openTempFile

Or am I missing something :thinking: ?

Reply:
`openTempFile` needs to be given a directory for the file. It's usually `/tmp` on Linux, but there are environment variables that control it. On Windows and MacOS, who knows what it might be. And we are going to want to run our database tests on these platforms.

`withSystemTempFile` and friends find the system temp directory for you.